drivers/led/sn32f2xx.c

```diff
@@ -380,7 +500,7 @@ static void update_pwm_channels(PWMDriver *pwmp) {
     for (uint8_t current_key_row = 0; current_key_row < MATRIX_ROWS; current_key_row++) {
         uint8_t led_index = g_led_config.matrix_co[current_key_row][current_key_col];
 #    if (SN32_PWM_CONTROL == SOFTWARE_PWM)
-        if (led_index >= SN32F24XB_LED_COUNT) continue;
+        if (led_index >= SN32F2XX_LED_COUNT) continue;
 #    endif
         uint8_t led_row_id = (current_key_row * SN32_RGB_MATRIX_ROW_CHANNELS);
         // Check if we need to enable RGB output
```

Comment:
In line 519 and 521 I need current_key_col  instead of last_key_col.
Otherwise the keypress effects are shifted one col to the left.

```
    // Enable RGB output
    if (enable_pwm_output) {
#        if (SN32_RGB_OUTPUT_ACTIVE_LEVEL == SN32_RGB_OUTPUT_ACTIVE_HIGH)
        writePinHigh(led_col_pins[current_key_col  ]);
#        elif (SN32_RGB_OUTPUT_ACTIVE_LEVEL == SN32_RGB_OUTPUT_ACTIVE_LOW)
        writePinLow(led_col_pins[current_key_col  ]);
#        endif // SN32_RGB_OUTPUT_ACTIVE_LEVEL
    }
```